control-plane-operator/controllers/hostedcontrolplane/ignition/reconcile.go

```diff
@@ -33,6 +33,7 @@ var (
 	defaultIgnitionConfigMapLabels = map[string]string{
 		"hypershift.openshift.io/core-ignition-config": "true",
 	}
+	kernelArgsMultipathForPowerVS = []string{"rd.multipath=default", "root=/dev/disk/by-label/dm-mpath-root"}
```

Comment:
```suggestion
	kernelArgsMultipathForPowerVS = []string{
				"rd.multipath=default", 
				"root=/dev/disk/by-label/dm-mpath-root",
			}
```

Minor cleanup. Rest looks good to me !